src/Endatix.Api/Endpoints/Submissions/ListByFormId.ListByFormIdValidator.cs

```diff
@@ -1,13 +1,29 @@
-﻿using FastEndpoints;
+﻿using Endatix.Api.Common;
+using FastEndpoints;
 using FluentValidation;
 
 namespace Endatix.Api.Endpoints.Submissions;
 
 public class ListByFormIdValidator : Validator<ListByFormIdRequest>
 {
+    private static readonly Dictionary<string, Type> _filterableFields = new()

```

Comment:
Yes, it's good to have in mind that defining the fields will evolve in the future. Here these are only for validation purposes. It's hard now to say which is the right place to get them defined (e.g. application, tenant, etc) as this will largely depend on the business requirements and needs. I would say we start with the most basic - the type has a set of filterable columns it is the direct owner and we evolve to more flexible approaches depending on the needs.